lib/ClimaLandSimulations/README.md

```diff
@@ -0,0 +1,16 @@
+# ClimaLandSimulations
+
+A library of methods to run ClimaLSM:
+- at single sites, such as FLUXNET
+- globally (WIP)
+
+## Installation
+
+```jl
+julia> ] # to go into pkg mode
+pkg> add ClimaLandSimulations
+pkg> *backspace* # press backspace button to go back into julia mode
+julia> using ClimaLandSimulations
+```
+
+For examples, see the [experiments](https://github.com/CliMA/ClimaLSM.jl/tree/ar/climalandlib/lib/ClimaLandSimulations/experiments) folder
```

Comment:
Im not 100% sure, but I think this link may need to update to (https://github.com/CliMA/ClimaLSM.jl/lib/ClimaLandSimulations/experiments) once we have merged this (or we might do it now)